cmd/root.go

```diff
@@ -135,7 +137,16 @@ var rootCmd = &cobra.Command{
 			os.Exit(1)
 		}
 		setupLog.Info("starting manager")
-		if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
+		if err := mgr.Start(
+			// HACK: pass down kube REST client config via context;
```

Comment:
This is the only mechanism I deemed viable at this point, please advise if can think of / want to have another solution.

Reply:
Can't we just create the clientSet interface within the provider? Like in https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/aws/auth/auth.go#L202-L205. Then you can use it again like this: https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/aws/auth/auth.go#L220-L224

Reply:
Oh interesting, I didn't know that this was possible without a manager instance, will try that.

Reply:
Changed.